vectors/vectors_cfrags.json

```diff
@@ -1,51 +1,51 @@
 {
   "name": "Test vectors for CFrags",
-  "description": "This is a collection of CFrags, originated from the enclosed Capsule, under the enclosed delegating, verifying and receiving keys. Each CFrag must deserialize correctly and can be replicated with a call to `pre.reencrypt(kfrag, capsule, provide_proof=False)`",
+  "description": "This is a collection of CFrags, originated from the enclosed Capsule, under the enclosed delegating, verifying and receiving keys. Each CFrag must deserialize correctly and can be replicated with a call to `pre.reencrypt(kfrag, capsule)`",
```

Comment:
The original goal of this test vector was to provide something completely deterministic and reproducible. Re-encryption is deterministic, but proving correctness is not, so that's why this test vector used the option `provide_proof=False`.

Having said that, our test vectors are incomplete, we don't have anything for correctness proofs, so I guess these ones can stay in this form or another.

Reply:
Hm, the way I understand it, *generating* a proof involves randomness, but once it is created and put in a vector, the rest is deterministic?

Reply:
Yes, you're right. But the point of this test vector is to be able to replicate re-encryption (and hence, production of CFrags). There's however value in having a test vector of valid correctness proofs, so maybe we can also keep these.